src/utils.ts

```diff
@@ -0,0 +1,41 @@
+import { DataViewerProposal } from './interfaces/DataViewerProposal';
+
+/**
+   * Safely gets the proposal value for a given proposal attribute key.
+   *
+   * @param  {DataViewerProposal} proposal
+   * @param  {string} key The key for the proposal attribute we want.
+   * @return {(Array|undefined)}
+   */
+const getProposalValue = (proposal: DataViewerProposal, key: string) => {
```

Comment:
This method used to have more checks in it, such as verifying `key in proposal.values`, until I realized (with the help of my test coverage) that they weren't necessary.

So in the end it's like… what does this do? Well, for now it removes those blank values. If the API promises not to send blank values, then I'm not entirely sure we do need it?